src/pytest_plugins/consume/consume.py

```diff
@@ -29,6 +29,45 @@
 )
 
 
+def print_migration_warning(terminalreporter: Any = None) -> None:
+    """Print migration warning about repository merge."""
+    lines = [
+        "",
+        "=" * 80,
+        "⚠️  IMPORTANT: Repository Migration in Progress - 'The Weld' ⚠️",
+        "=" * 80,
+        "",
+        "This repository is being merged into ethereum/execution-specs during the",
```

Comment:
We mention EELS below but never define it. We can either add this change here or change all EELS in this note to `ethereum/execution-specs` to be more explicit. wdyt?

```suggestion
        "This repository is being merged into ethereum/execution-specs (EELS) during the",
```